client/ayon_maya/plugins/publish/extract_maya_scene_raw.py

```diff
@@ -79,7 +78,7 @@ def process(self, instance):
         if set(self.add_for_families).intersection(
                 set(instance.data.get("families", []))) or \
                 instance.data.get("productType") in self.add_for_families:
-            selection += self._get_loaded_containers(members)
+            selection += get_container_members(members)
```

Comment:
This change is wrong.. we still need to first get the containers that are members of this instance... and then go from there. The logic you have here now will error.